backend/src/modules/doubles-tournament/core/application/query/FindMatchesQueueByTournamentId.ts

```diff
@@ -0,0 +1,11 @@
+import { MatchesQueue } from '../../domain/MatchesQueue';
+
+export class FindMatchesQueueByTournamentById {
```

Comment:
nie żebym się czepiał, ale to drugie 'By' jest tutaj zbędne ;)

Reply:
no trochę za hojnie tutaj poleciałem z tymi "By" :-P - copy, paste chyba za bardzo wjechał